packages/react-next/src/Utilities/Customizer.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {
+  Customizations,
+  CustomizerContext,
+  mergeCustomizations,
+  ICustomizerContext,
+} from 'office-ui-fabric-react/lib/Utilities';
+import { ThemeProvider } from '@fluentui/react-theme-provider';
+import { convertLegacyTheme } from '../Styling/convertLegacyTheme';
+
+import { ICustomizerProps as LegacyCustomizerProps } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface ICustomizerProps extends LegacyCustomizerProps {
+  disableThemeProvider?: boolean;
```

Comment:
weird boolean

Reply:
it's not ideal, but it's necessary to avoid `ThemeProvider` when it's not needed. couple scenarios:
1. theme updated with only RTL flipping (see the use case in `Fabric`)
2. when `Customizer` is used ONLY on components that are still not converted to scss yet.